src/main/groovy/life/qbic/datamodel/dtos/business/ProductCategory.groovy

```diff
@@ -15,7 +15,8 @@ enum ProductCategory {
     SECONDARY_BIOINFO("Secondary Bioinformatics"),
     DATA_STORAGE("Data Storage"),
     PROTEOMIC("Proteomics"),
-    METABOLOMIC("Metabolomics")
+    METABOLOMIC("Metabolomics"),
+    EXTERNAL_SERVICE("External Service")
```

Comment:
Do we want to distinguish an external service on the product level? I think it makes more sense to do it solely based on the facility of the product. It seems very arbitrary to me to introduce an external service which would be e.g. Sequencing.

Reply:
It is indeed generic by design, after talking to the stakeholders. we dont know what an external service will be, and we dont need to.